xformers/ops/fmha/_triton/splitk_kernels.py

```diff
@@ -341,7 +341,8 @@ def _fwd_kernel_splitK(
     # scale sm_scale by log_2(e) and use
     # 2^x instead of exp in the loop because CSE and LICM
     # don't work as expected with `exp` in the loop
-    qk_scale = sm_scale * 1.44269504
+    log2e = tl.full((), 1.44269504, tl.float32)
```

Comment:
Can we add a comment with an explanation/reference for why the code is this way?

Reply:
Yes definitely, good call